### ISSUE TITLE
Adicionar campo de estado do paciente

### ISSUE THREAD

**Issue by barbosaaob** [LED-UFAL-Covid-19/smc19#6]
Adicionar campo para informar o estado de um paciente. As opções são:
- testado
- suspeito
- confirmado
- imune
- morto

**Comment by jhonnye0** [comment]
No Model do Profile existe um campo, **status** com as seguintes _choices_:

 status = (
    ('N', 'Normal'),
    ('S', 'Suspeito'),
    ('C', 'Confirmado'),
    ('M', 'Morto'),
    ('R', 'Recuperado')
)

O que falta é poder editar esse campo, na edição do profile de um paciente eu posso incluir editar esse campo.

**Comment by jhonnye0** [comment]
Além disso devo incluir ou remover quais estados?

**Comment by jhonnye0** [comment]
Na verdade vou tentar corrigir o porquê desse status nao estar aparecendo

**Comment by barbosaaob** [comment]
precisamos de mais estados:
- hospitalizado
- UTI

isso tb precisa aparecer no formulário de atendimento.

**Comment by barbosaaob** [comment]
parametrizar o script que gera dados para adicionar casos de hospitalização e uti.

**Comment by valeriojr** [comment]
então é pra adicionar os dois estados e deixar visiveis. É pra habilitar a edição também?

**Comment by barbosaaob** [comment]
as coias se misturaram aqui.

vamos precisar do campo do resultado do exame que só pode ser alterado pela sec de saúde. esse tem as opções:
- testado ou aguardando resultado ou sem resultado
- positivo (confirmado)
- negativo

além disso, precisamos de um estado do paciente que pode ser alterado pelo atendente com os seguintes estados:
- hospitalizado
- uti
- morto

além dos campos de suspeito e testado que já temos no formulário. esses últimos também podem ser alterados pelo atendente.

**Comment by valeriojr** [comment]
Atualmente a lista de estados é essa:
```python
status = (
    ('N', 'Normal'),
    ('T', 'Testado'),
    ('S', 'Suspeito'),
    ('C', 'Confirmado'),
    ('M', 'Morto'),
    ('I', 'Imune'),
    ('H', 'Hospitalizado'),
    ('U', 'UTI')
)
```
Essa variável fica guardada no profile. O @jhonnye0 tinha feito a parte da alteração do estado ontem. Se não foi removido, já deve aparecer com os novos valores

**Comment by barbosaaob** [comment]
> as coias se misturaram aqui.
> 
> vamos precisar do campo do resultado do exame que só pode ser alterado pela sec de saúde. esse tem as opções:
> 
> * testado ou aguardando resultado ou sem resultado
> * positivo (confirmado)
> * negativo
> 
> além disso, precisamos de um estado do paciente que pode ser alterado pelo atendente com os seguintes estados:
> 
> * hospitalizado
> * uti
> * morto
> 
> além dos campos de suspeito e testado que já temos no formulário. esses últimos também podem ser alterados pelo atendente.

como eu disse aqui, essa variável ta misturando tudo. as coisas precisam ser separadas e algumas tem restrições de perfil pra atender.

**Comment by valeriojr** [comment]
Esse status é alterado quando um novo atendimento é feito ou editado. Ex:
Se o atendimento foi marcado como suspeito e não positivo, então o usuário é marcado como suspeito. Se o atendimento foi marcado com a resposta positivo o status vira positivo e por aí vai, pra sempre pegar o status mais atual na hora de fazer o mapa/estatísticas. Eu deveria criar outro campo para esses outros estados (UTI, hospitalizado)? Ou eles também vão entrar para as estatísticas?

**Comment by barbosaaob** [comment]
> Esse status é alterado quando um novo atendimento é feito ou editado. Ex:
> Se o atendimento foi marcado como suspeito e não positivo, então o usuário é marcado como suspeito. Se o atendimento foi marcado com a resposta positivo o status vira positivo e por aí vai, pra sempre pegar o status mais atual na hora de fazer o mapa/estatísticas. Eu deveria criar outro campo para esses outros estados (UTI, hospitalizado)? Ou eles também vão entrar para as estatísticas?

são 3 informações independentes: resultado do exame, testado/suspeito e hospitalizado/uti. pra ter resultado do exame tem que ser testado, mas não precisa necessariamente ter sido considerado suspeito antes no formulário pq podem considerar isso desnecessário. estar na uti ou hospitalizado também não depende dos outros necessariamente.

**Comment by valeriojr** [comment]
então deixa eu ver se entendi, o Profile tem que ter os seguintes campos:
- Resultado do exame (**Sem resposta**, **Negativo**, **Positivo**)
- **Testado** ou **suspeito** ou **nada** (não encontrei um nome pra essa variável)
- Estado (**Não está internado**, **Internado**, **Internado na UTI**, **Morto**)
?

**Comment by valeriojr** [comment]
Certo, então eu vou tirar o status do Profile e usar o annotate pra pegar o mais recente. A vantagem é que vai poder dizer a data de referência também, aí dá pra calcular o aumento desde uma determinada data
